src/math.cc

```diff
@@ -129,17 +135,18 @@ extern "C"
     return ::pow(x, y);
   }
 
-  bool isnanf(float x)
+  int isnanf(float x)
```

Comment:
> Changing the return types makes sense to me

I like that way of putting it. Indeed, I have struggled a lot in over the years on these kinds of points.

When wrapping the STL for `<cmath>` it seems that a certain amount of _empirical_ observation will always be necessary. This is because the compiler's `<math.h>` header file should be taken and left unchanged. The compiler supplier, however, will be advancing this C header and (hopefully) gettning closer and closer to standards adherence as we progress.

The bottom line is we need to keep an eye on the evolution of `<math.h>` and ensure that this is properly paired with `avr-libstdcpp`.